tests/Unit/FormRequests/StoreNewRegistrationRequestTest.php

```diff
@@ -0,0 +1,227 @@
+            'requestShouldFailWhenEligibilityIsMissing' => [
+                'passed' => false,
+                'data' => [
+                    'consent' => 'on',
+                    'carer' => 'A String',
+                ]
+            ],
+            'requestShouldFailWhenCarerIsNotAString' => [
+                'passed' => false,
+                'data' => [
+                    'consent' => true,
+                    'carer' => 1,
+                    'eligibility' => 'other'
+                ]
+            ],
+            'requestShouldFailWhenEligibilityIsNotInEnum' => [
+                'passed' => false,
+                'data' => [
+                    'consent' => 'yes',
+                    'carer' => 'A String',
+                    'eligibility' => 'hello'
+                ]
+            ],
+            'requestShouldFailWhenConsentIsNotTruthy' => [
+                'passed' => false,
+                'data' => [
+                    'consent' => 2,
+                    'carer' => 'A String',
+                    'eligibility' => 'healthy-start'
+                ]
+            ],
+            'requestShouldSucceedWithSecondaryCarers' => [
+                'passed' => true,
+                'data' => [
+                    'consent' => 1,
+                    'carer' => 'A String',
+                    'eligibility' => 'other',
+                    'carers' => ['B String', 'C String'],
+                ]
+            ],
+            'requestShouldFailWithEmptySecondaryCarers' => [
+                'passed' => false,
+                'data' => [
+                    'consent' => 1,
+                    'carer' => 'A String',
+                    'eligibility' => 'other',
+                    'carers' => [],
+                ]
+            ],
+            'requestShouldFailWithNonStringSecondaryCarers' => [
+                'passed' => false,
+                'data' => [
+                    'consent' => 1,
+                    'carer' => 'A String',
+                    'eligibility' => 'other',
+                    'carers' => [1,2,3,4],
+               ]
+            ],
+            'requestShouldSucceedWithMinimalChildren' => [
+                'passed' => true,
+                'data' => [
+                    'consent' => 1,
+                    'carer' => 'A String',
+                    'eligibility' => 'other',
+                    'children' => [
+                        0 => ['dob' => '2017-09']
+                    ],
+                ]
+            ],
+            'requestShouldFailWithChildInvalidDobFormat' => [
+                'passed' => false,
+                'data' => [
+                    'consent' => 1,
+                    'carer' => 'A String',
+                    'eligibility' => 'other',
+                    'children' => [
+                        0 => ['dob' => '2017-09-01']
+                    ],
+                ]
+            ],
+            'requestShouldFailWithEmptyChildren' => [
```

Comment:
Is this true?

Reply:
Actually, on reflection, yes

because it's a form input post, there will be no children array at all if no children are added.

Reply:
An array being present must have one member